public/js/interface.js

```diff
@@ -0,0 +1,134 @@
+const game = new Game();
+let frame = new Frame();
+let i = 0;
```

Comment:
This could be more descriptively named - 'i' doesn't really communicate what this variable is supposed to represent.  Maybe it could be called frameCount or something similar.

Reply:
Hey Sam! Thanks a lot for your review again!
The variable 'i' represents an accumulator in both cases, a tricky helper to allow me to generate pins (first case) and hide them as they are knocked (second case).
In the second case it helps me in a for loop to iterate through pin and hide them until the pin reaches the id num of the knocked one.

I have renamed 'i' in both cases following your suggestion as follows:
1- **pinNum**
2- **pinIterator**